x/evm/keeper/integration_test.go

```diff
@@ -476,3 +469,28 @@ var _ = Describe("Handling a MsgEthereumTx message", Label("EVM"), Ordered, func
 		})
 	})
 })
+
+// setupIntegrationTestSuite is a helper function to setup a integration test suite
+// with a network with a specified custom genesis state
+func setupIntegrationTestSuite(customEVMGenesis *evmtypes.GenesisState) *IntegrationTestSuite {
+	customGenesis := network.CustomGenesisState{
+		evmtypes.ModuleName: customEVMGenesis,
+	}
+
+	keyring := testkeyring.New(3)
+	integrationNetwork := network.New(
+		network.WithPreFundedAccounts(keyring.GetAllAccAddrs()...),
+		network.WithCustomGenesis(customGenesis),
+	)
+	grpcHandler := grpc.NewIntegrationHandler(integrationNetwork)
+	txFactory := factory.New(integrationNetwork, grpcHandler)
+
+	suite := &IntegrationTestSuite{
+		network:     integrationNetwork,
+		factory:     txFactory,
+		grpcHandler: grpcHandler,
+		keyring:     keyring,
+	}
+
+	return suite
+}
```

Comment:
I think with the new update params functions this should not be needed anymroe and params could be updated through proposal